src/utils/getENS.ts

```diff
@@ -0,0 +1,9 @@
+import { ENSAddressPair } from '@types';
+
+export function getENS(address: string, ENSRegistry: ENSAddressPair[]): ENSAddressPair[] {
```

Comment:
why does this return an array if there can only be one ENS name per address?

Reply:
Resolved, by using your recommended type